OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -93,6 +93,9 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Sound to play when the projectile hits the ground, but not the target.")]
 		public readonly string BounceSound = null;
 
+		[Desc("Terrain where the projectile explodes instead of bouncing.")]
+		public readonly string[] InvalidBounceTerrain = null;
```

Comment:
```suggestion
		public readonly HashSet<string> InvalidBounceTerrain = new HashSet<string>();
```

seems to be the standard in the current code base. I guess because of performance.

Reply:
Done.